library/src/lib/calendar/calendar-header/calendar-header.component.html

```diff
@@ -0,0 +1,40 @@
+<header class="fd-calendar__header">
+    <div class="fd-calendar__navigation">
+        <div class="fd-calendar__action">
+            <button class="fd-button--standard fd-button--light fd-button--compact sap-icon--slim-arrow-left"
+                    [attr.id]="id + '-left-arrow'"
+                    [attr.aria-label]="previousLabel"
+                    [attr.aria-disabled]="false"
+                    type="button"
+                    (click)="previousClicked.emit()">
+            </button>
+        </div>
+        <div class="fd-calendar__action">
+            <button class="fd-button--light fd-button--compact"
+                    [attr.aria-label]="calendarI18nLabels.monthSelectionLabel"
+                    [attr.aria-selected]="activeView === 'month'"
```

Comment:
Logic like this one where the boolean expression is not dependent on an argument should be done by a get method